### DISCUSSION TITLE
Question about structure of the docs

### DISCUSSION THREAD

**Discussion by bepnos** [LazyVim/LazyVim#1310] [Q&A]
This is most likely a very noob question but Im having trouble understanding the docs exactly, for example on the "Eslint" page it has 1 line stating "To use this, add it to your lazy.nvim imports" with some lines after it. Okay simple enough, however after this there is a subtitle "nvim-lspconfig" with some code: "option" and "full-spec". Do I have to do anything with this ? Do I have to create a new .lua file with this name and paste the code ? Are these default settings ? What is the difference between "options" and "full-spec" ? If I need to paste these lines anywhere where exactly do I need to paste these and in what file ? This is in several pages where there are just headers of a package name but I dont know if Im expected to paste these lines anywhere and if so where.

**Comment by dpetka2001** [comment]
For the extras you only have to put the `import` line in `/config/lazy.lua`. The rest of the page describes what packages are bundled together and their configuration to make the extra work. If you don't import the extra, you would then have to do all the configuration that's described in the rest of the page by yourself.

**Reply by bepnos** [reply]
And what abouts pages that simply have these extras for example the TreeSitter page only has this very long file what do I need to add to make TreeSitter work ?

**Reply by dpetka2001** [reply]
Those are just the default configurations that come with Lazyvim. Treesitter will just work with the options you see in that page as the default configuration. You only need to make changes to the configuration if you want some other kind of behavior that the default Lazyvim doesn't support.
